### PR TITLE
53 adapt new base design

### PR DESCRIPTION
**Pull request by SebastianMaier03**

### CODE REVIEW

File: app/selectLayout.css [hunk 1]
```diff
@@ -4,4 +4,6 @@
     border-radius: 1rem;
     padding: 2rem 3rem;
     height: 100%;
+
```
Comment (DorinaSobiecki) [inline app/selectLayout.css:7]: Delete empty line

File: app/utils/colorUtils.tsx [hunk 2]
```diff
@@ -25,8 +25,21 @@ const rocketColor: Color[] = [
     { code: "#7A41F3", word: "purple" },
     { code: "#FFE500", word: "yellow" }
 ];
+
+const getHairColorIndex = (colorCode: string): number => {
+    return hairColors.findIndex(color => color.code === colorCode);
+};
+
+const getSKinColorIndex = (colorCode: string): number => {
```
[the remark this answers is not in the document]
Comment (SebastianMaier03) [reply app/utils/colorUtils.tsx]: Fixed

File: pages/character.tsx [hunk 3]
```diff
@@ -49,94 +65,104 @@ const Character: React.FC = () => {
...
         const colorWord = getSkinColors()[index].code;
         setSelectedSkinColorCode(colorWord);
+        setSelectedIndexSkinColor(index)
         if (isClient) await setSkinColor(colorWord);
     };
 
     const renderLeftChildren = () => (
-        <div className="h-full w-full flex justify-center items-center">
-            <CharacterImage hairColor={selectedHairColorCode}  hairType={selectedHair} skinColor ={selectedSkinColorCode}></CharacterImage>
+        <div className="character-image h-full w-full flex justify-center items-center">
```
Comment (DorinaSobiecki) [inline pages/character.tsx]: character-image is no longer needed

**Review by DorinaSobiecki — CHANGES_REQUESTED** [review]
Take a look at the comments

**Review by DorinaSobiecki — APPROVED** [review]
LGTM

**Pull request merged**